helm/superset/values.schema.json

```
@@ -0,0 +1,590 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "replicaCount": {
+            "type": "integer"
+        },
+        "runAsUser": {
+            "type": "integer"
+        },
+        "serviceAccount": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "create": {
+                    "type": "boolean"
+                }
+            },
+            "required": [
+                "create"
+            ]
+        },
+        "bootstrapScript": {
+            "type": "string"
+        },
+        "configFromSecret": {
+            "type": "string"
+        },
+        "envFromSecret": {
+            "type": "string"
+        },
+        "envFromSecrets": {
+            "type": "array"
+        },
+        "extraEnv": {
+            "type": "object"
+        },
+        "extraEnvRaw": {
+            "type": "array"
+        },
+        "extraSecretEnv": {
+            "type": "object"
+        },
+        "extraConfigs": {
+            "type": "object"
+        },
+        "extraSecrets": {
+            "type": "object"
+        },
+        "extraVolumes": {
+            "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.23.0/_definitions.json##/definitions/io.k8s.api.apps.v1.PodSpec/properties/volumes"
+        },
+        "extraVolumeMounts": {
+            "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.23.0/_definitions.json##/definitions/io.k8s.api.apps.v1.PodSpec/properties/volumeMounts"
+        },
+        "configOverrides": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+                "extend_timeout": {
+                    "type": "string"
+                },
+                "enable_oauth": {
+                    "type": "string"
+                }
+            }
```

```suggestion
            "additionalProperties": true,
            "properties": {
                "type": "object",
                "properties": {
                    "type": "string"
                }
            }
```
We support many configuration options, the set of possible values is determined by the superset version. To be future-proof, I propose to give us full freedom here.

---

I don't think that it gonna work in the way you will expect. You rather need:

```json
"additionalProperties": {
    "type": "string"
},
``` 

In a moment I will commit that fix, so let me know what do you think.

---

Overall I fully agree that `additionalProperties` should be enabled there.